javascript/javascript1/week3/Series-duration-of-my-life.js

```diff
@@ -0,0 +1,54 @@
+const seriesDurations = [
+  {
+    title: "Game of thrones",
+    days: 3,
+    hours: 1,
+    minutes: 0,
+  },
+  {
+    title: "Just add magic",
+    days: 0,
+    hours: 22,
+    minutes: 30,
+  },
+  {
+    title: "Just add magic:Mystory city",
+    days: 0,
+    hours: 4,
+    minutes: 30,
+  },
+];
+var avgLifeSpan = 80;
+const hoursInDay = 24;
+const daysInYear = 365.5;
+//console.log('soujanya');
+console.log(seriesDurations);
+//console.log(seriesDurations[0].title);
+var durationOfGameOfThronesInHours= (seriesDurations[0].days * hoursInDay ) + seriesDurations[0].hours ;
+//console.log(durationOfGameOfThronesInHours);
+//console.log(seriesDurations[1].title);
+var durationOfJustAddMagicInHours= (seriesDurations[1].days * hoursInDay ) + seriesDurations[1].hours + (seriesDurations[1].minutes/60) ;
+//console.log(durationOfJustAddMagicInHours);
+//console.log(seriesDurations[2].title);
+var durationOfMystoryCityInHours= (seriesDurations[2].days * hoursInDay ) + seriesDurations[2].hours + (seriesDurations[2].minutes/60) ;
+//console.log(durationOfMystoryCityInHours);
+logOutSeriesText();
+
+function logOutSeriesText() {
+var avgLifeSpanInHours = daysInYear * hoursInDay * avgLifeSpan;
```

Comment:
same

Reply:
ok.Thankyou for your comments, will update soon